### PR TITLE
Relay data fixes for Chooser dialogs

### PR DESCRIPTION
**Pull request by keithpitt**

### CODE REVIEW

File: app/components/member/Teams/chooser.js [hunk 1]
```diff
@@ -98,25 +100,23 @@ class Chooser extends React.Component {
...
-          showingDialog: true
-        }, () => {
-          this._autoCompletor.focus();
-        });
+        this.setState({ loading: false });
       }
     });
+
+    // Now start showing the dialog, and when it's open, autofocus the first
+    // result.
```
Comment (ticky) [inline app/components/member/Teams/chooser.js]: “the first result” doesn’t seem to be what this does?

File: app/components/member/Teams/chooser.js [hunk 2]
```diff
@@ -168,7 +168,7 @@ class Chooser extends React.Component {
 
 export default Relay.createContainer(Chooser, {
   initialVariables: {
-    showingDialog: false,
+    isMounted: false,
```
Comment (ticky) [inline app/components/member/Teams/chooser.js]: Why did you change this to `isMounted`? This _doesn’t_ occur when the dialog is mounted

File: app/components/member/Teams/chooser.js (hunk 1 above)
Comment (keithpitt) [inline app/components/member/Teams/chooser.js]: Lol, this is a bad comment - I meant to say focus the search input. Will fix my comments!

File: app/components/member/Teams/chooser.js (hunk 2 above)
Comment (keithpitt) [inline app/components/member/Teams/chooser.js]: Oh you're totally right, I was rushing through some last minute changes and this is poorly named. I'll change them all back to `showingDialog` which is actually what it _is_
